packages/bpk-mixins/src/mixins/_chips.scss

```diff
@@ -24,158 +24,215 @@ $bpk-spacing-v2: true;
-      background-color: $bpk-color-sky-gray-tint-04;
+      @include bpk-border-sm($bpk-core-primary-day);
     }
 
     &-selected {
       @include bpk-themeable-property(
         color,
-        --bpk-chip-primary-selected-text-color,
-        $bpk-color-white
+        --bpk-chip-default-selected-text-color,
+        $bpk-text-on-dark-day
       );
       @include bpk-themeable-property(
         background-color,
-        --bpk-chip-primary-selected-background-color,
-        $bpk-color-sky-blue
+        --bpk-chip-default-selected-background-color,
+        $bpk-core-primary-day
       );
       @include bpk-border-sm(transparent);
 
       @include bpk-hover {
-        @include bpk-themeable-property(
-          background-color,
-          --bpk-chip-primary-selected-hover-background-color,
-          $bpk-color-sky-blue-shade-01
-        );
+        @include bpk-border-sm(transparent);
       }
 
       &:active:not(:disabled) {
-        @include bpk-themeable-property(
-          background-color,
-          --bpk-chip-primary-selected-active-background-color,
-          $bpk-color-sky-blue-shade-02
-        );
+        @include bpk-border-sm(transparent);
+      }
+    }
+
+    &-dismissible {
+      @include bpk-hover {
+        svg {
+          fill: currentColor;
+        }
+      }
+
+      &:active:not(:disabled) {
+        svg {
+          fill: currentColor;
+        }
+      }
+
+      &__trailing-accessory-view {
+        fill: $bpk-text-disabled-on-dark-day;
       }
     }
+
+    &-disabled {
+      @include bpk-border-sm(transparent);
+    }
   }
 
-  &--success {
-    background-color: $bpk-color-sky-gray-tint-06;
-    color: $bpk-color-sky-gray;
+  &--on-image {
+    background-color: $bpk-surface-default-day;
+    color: $bpk-text-primary-day;
+
+    @include bpk-box-shadow-sm;
 
     @include bpk-hover {
-      background-color: $bpk-color-sky-gray-tint-05;
+      background-color: $bpk-canvas-contrast-day;
     }
 
     &:active:not(:disabled) {
-      background-color: $bpk-color-sky-gray-tint-04;
+      background-color: $bpk-canvas-contrast-day;
     }
 
     &-selected {
       @include bpk-themeable-property(
         color,
-        --bpk-chip-success-selected-text-color,
-        $bpk-color-sky-gray
+        --bpk-chip-on-image-selected-text-color,
+        $bpk-text-on-dark-day
       );
       @include bpk-themeable-property(
         background-color,
-        --bpk-chip-success-selected-background-color,
-        $bpk-color-glencoe
+        --bpk-chip-on-image-selected-background-color,
+        $bpk-core-primary-day
       );
 
       @include bpk-hover {
-        // There's no suitable token here, so we use darken.
         @include bpk-themeable-property(
           background-color,
-          --bpk-chip-success-selected-hover-background-color,
-          darken($bpk-color-glencoe, 10%)
+          --bpk-chip-on-image-selected-hover-background-color,
+          $bpk-core-primary-day
```

Comment:
is this needed? would it otherwise inherit the background colour set in the `on-image` mixin hover state? As in that case we would need to do the same for the other styles `default` and `on dark`

Reply:
Yes, this is necessary. If not set, the hover background color after being selected will inherit the hover background color before being selected.
The case of default and on dark is different from on image.
In the case of on image, the background color of the Chip component will change when the hover state is not selected. But in the case of default or on dark, the only thing that changes in the unchecked hover state is the border color, the background color does not change.

Reply:
You're right! Makes sense 🙂